src/Illuminate/Auth/Guard.php

```diff
@@ -180,7 +180,7 @@ public function id()
             return;
         }
 
-        $id = $this->session->get($this->getName(), $this->getRecallerId());
+        $id = $this->session->get($this->getName());
```

Comment:
`$this->getRecallerId()` get user id from cookie，but it did not check whether the cookie's token is valid